lesson_002/01_circle.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Есть значение радиуса круга
+from math import pi, sqrt
+
+
+radius = 42
+pi # АНТОН: Пи я спициально заимпортил из библиотечки math, как и функцию sqrt.
+
+# Выведите на консоль значение прощади этого круга с точностю до 4-х знаков после запятой
+# подсказки:
+#       формулу можно подсмотреть в интернете,
+#       пи возьмите равным 3.1415926
+#       точность указывается в функции round()
+# TODO здесь ваш код
+
+print (round (pi * (float (radius) ** 2), 4))
+
+
+# Далее, пусть есть координаты точки
+point = (23, 34)
+# где 23 - координата х, 34 - координата у
+
+# Если точка point лежит внутри того самого круга (radius = 42), то выведите на консоль True,
+# Или False, если точка лежит вовне круга.
+# подсказки:
+#       нужно определить расстояние от этой точки до начала координат (0, 0)
+#       формула так же есть в интернете
+#       квадратный корень - это возведение в степень 0.5
+#       операции сравнения дают булевы константы True и False
+# TODO здесь ваш код
+
+point_length = sqrt((point[0]**2) + (point[1]**2))
+print (point_length < float (radius))
```

Comment:
No need to explicitly cast `radius` to float. Interpreter will do this automatically.